src/app/store/recipe.effect.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@angular/core';
+import { Actions, createEffect, ofType } from '@ngrx/effects';
+import { catchError, exhaustMap, map, mergeMap, of, switchMap } from 'rxjs';
+import { RecipesService } from '../recipes/services/recipes-service';
```

Comment:
same as above so you can use `@services/`
btw change your folder structure, `services` folder should be in separate folder in app/ and you put all services there (in separate folders)